Projects/hCoreUI/Sources/hForm/hRow.swift

```diff
@@ -139,6 +164,12 @@ extension hRow {
         }
     }
 
+    public var withNewChevronAccessory: hRow<Content, ChevronNewAccessory> {
```

Comment:
I think you can ditch this after you merge ChevronNewAccessory and ChevronAccessory